lte/gateway/python/magma/pipelined/app/gy.py

```diff
@@ -137,8 +137,9 @@ def _deactivate_flows_for_subscriber(self, imsi, ip_addr):
             self._datapath,
             imsi,
         )
-        self._qos_mgr.remove_subscriber_qos(imsi)
-        self._remove_he_flows(ip_addr, None)
+        if self._qos_mgr:
+            self._qos_mgr.remove_subscriber_qos(imsi)
+        self._remove_he_flows(ip_addr, "")
```

Comment:
This line is unrelated, but I saw in the IDE that the type signature is a string, not an optional string.

Reply:
good catch.